contrib/rosetta/configuration/staking.ros

```diff
@@ -91,7 +91,12 @@ staking(3){
       {
         "operation_identifier":{"index":2},
         "type":"cosmos.staking.v1beta1.MsgDelegate",
-        "account": {{recipient}},
+        "account": {
+            "address": {{recipient.address}},
```

Comment:
I think we mentioned here to use some dummy string? Like module_account or whatever?

Reply:
ah yes missed this, updating

Reply:
If not, technically it looks like you send this amount to yourself.